### ISSUE TITLE
Make release

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-io#66]
@KateMoreva @constantinpape 
I am working on a release today. Keeping track here.
I updated some dependencies in the `develop` branch and (accidentally) also already brought the `main` branch to the same state.

**Comment by tischi** [comment]
I updated MoBIE-beta. 
Could you please check whether one can open the v04 OME-Zarr and also whether the rest is working.

**Comment by constantinpape** [comment]
I just checked with the new MoBIE-beta and everything works as expected.

**Comment by tischi** [comment]
Should we wait with the non-beta release until the writer for v0.4 works?

**Comment by tischi** [comment]
I pushed the most recent versions to MoBIE-beta, but I did not test anything manually (apart from the automatic tests).

"Does it work"? Whatever you do there, could we add it as an automated test?

**Comment by tischi** [comment]
@constantinpape @K-Meech 

I would then make a MoBIE (non-beta) release with what we have right now.
OK?

**Comment by constantinpape** [comment]
@tischi sorry, I overlooked your previous message. I will test two more things and then let you know.
Could you maybe test writing something from the GUI?

**Comment by tischi** [comment]
@constantinpape 
Yes, it is here: `/g/cba/exchange/ome-zarr/2022/xyzct_8bit__mobie-2-1-0.ome.zarr`

**Comment by K-Meech** [comment]
@tischi Could you also update MoBIE-beta again when you do this? I added a tiny fix to the develop branch that needs to go there. (otherwise I can do it :)

> Could you maybe test writing something from the GUI?

@constantinpape For tests - there are a bunch for adding images  from the API, but not via the GUI. It's a bit difficult to test, as it needs a bunch of interaction with various dialogs... I'm not sure how it could be done.

**Comment by constantinpape** [comment]
> @constantinpape For tests - there are a bunch for adding images from the API, but not via the GUI. It's a bit difficult to test, as it needs a bunch of interaction with various dialogs... I'm not sure how it could be done.

I think that's fine, but we should then still manually check once that it works via the GUI in the beta release. (and I don't know how, I will check the reading part via the GUI again)

**Comment by constantinpape** [comment]
@tischi I checked now and opening ome.zarr files and MoBIE projects works as expected.

I am still confused about this: https://github.com/mobie/mobie-io/issues/66#issuecomment-1043029365

**Comment by K-Meech** [comment]
> we should then still manually check once that it works via the GUI in the beta release. (and I don't know how, I will check the reading part via the GUI again)

I've tested adding a few images via the GUI on the current MoBIE-beta, and all looks good to me. The only part that doesn't work, is the one that is resolved by the fix on the develop branch.

**Comment by constantinpape** [comment]
All releases are done *hooray*
